### PR TITLE
Fixing positioning when slim sidebars is enabled

### LINKED ISSUE #313: block tracker flickering with slim sidebar
from the ask box:
>Just a minor thing - after the update to blogtracker, the numbers have started jumping/flickering around when the mouse moves over them. I've got slim sidebar on, if that matters, and I'm on Chromium (so, basically Chrome, I guess).

followup to #311 @bvtsang

### PR DESCRIPTION
**Pull request by blackjackkent**
Fixes #313

### CODE REVIEW

**Comment by bvtsang** [comment]
Although this fixes the issue, I'd like to make sure that

- The CSS looks good on both slimmed and non-slimmed versions
- We preserve the behavior of the ✖ bumping the count to the left

**Comment by blackjackkent** [comment]
Are you not seeing that result? I believe it is acting the same in both scenarios.

**Comment by nightpool** [comment]
image
are the numbers supposed to look like that?

also the numbers don't move they stay in the same place no matter if the x is there or not. I think this is a good behavior, but maybe its unintentional?

**Comment by blackjackkent** [comment]
Per @bvtsang comment above we want to retain the movement behavior. I've updated the PR accordingly.

**Comment by bvtsang** [comment]
@nightpool The numbers aren't supposed to look like that. They should be aligned with the name and ✖ button.

To realign the count, we may want to change some of the CSS in tweaks. If we do, we'll need to test not just the slimmed down blog tracker but the slimmed down tag tracker as well.

**Comment by blackjackkent** [comment]
My most recent update should handle that positioning better and align everything.

**Comment by bvtsang** [comment]
By any chance, did you forget to push your changes? I don't see them yet.

**Comment by blackjackkent** [comment]
No, ce37dfd above should handle the alignment:

align1
align2

**Comment by bvtsang** [comment]
Looks better! Can we move the counts and close buttons a bit downward toward the center of each row?

**Comment by nightpool** [comment]
yeah I feel like the implied solution to that was to center the counts, not
offset the x button :D

**Comment by blackjackkent** [comment]
*tries again cautiously* :D

File: Extensions/people_notifier.css [hunk 1]
```diff
@@ -21,20 +27,25 @@
 
 .xkit-people-notifier-person a:after {
 	background: none !important;
+	display: block;
+	height: 100%;
+	position: relative;
 }
 
-.xkit-people-notifier-person .count {
+.identity .xkit-people-notifier-person .controls_section a .count,
```
Comment (bvtsang) [inline Extensions/people_notifier.css]: Swap the order of `.xkit-people-notifier-person` and `.controls_section`.

File: Extensions/people_notifier.css [hunk 2]
```diff
@@ -21,20 +27,25 @@
...
+.xkit-people-notifier-person .close {
 	float: right;
-	position: relative !important;
-	top: -16px !important;
+        top: 50% !important;
+        display: block;
+        height: 13px;
+        margin-top: -13px;
```
Comment (bvtsang) [inline Extensions/people_notifier.css]: Try `-11px`.

File: Extensions/people_notifier.css [hunk 3]
```diff
@@ -21,20 +27,25 @@
...
+	position: relative;
 }
 
-.xkit-people-notifier-person .count {
+.identity .xkit-people-notifier-person .controls_section a .count,
+.xkit-people-notifier-person .close {
 	float: right;
-	position: relative !important;
-	top: -16px !important;
+        top: 50% !important;
```
Comment (bvtsang) [inline Extensions/people_notifier.css]: Mixed spaces and tabs within this block.

File: Extensions/people_notifier.css [hunk 4]
```diff
@@ -5,6 +5,12 @@
 	color: rgba(255,255,255,0.55);
 }
 
+.xkit-people-notifier-person a {
```
Comment (bvtsang) [inline Extensions/people_notifier.css]: Nitpick: convert to tabs

**Comment by blackjackkent** [comment]
Good call. Updated.

**Comment by bvtsang** [comment]
LGTM

@homu r+

**Pull request merged**